doc/source/cost.rst

```diff
@@ -0,0 +1,45 @@
+Estimating costs
+================
+
+It is difficult to estimate costs exactly, because this depends highly on your
+particular setup. In general, there are a few primary factors that control
+cost:
+
+1. What kind of hardware do your users need?
+2. How many users do you have?
+3. What usage pattern will your users have?
+
+Hardware
+--------
+The largest driver of cost in hardware is memory (RAM). More RAM means that your
+users will be able to work with larger datasets with more flexibility, but it
+can also be expensive. As a general rule, costs associated with RAM scale
+at <XXX> cost.
+
+Another consideration is whether to allow your users to have persistent storage.
+If users don't have persistant storage, then disks will be wiped once users
+are finished with their sessions. None of their changes will be saved. This
+requires significantly fewer storage resources, and also results in faster
+load times. Storage roughly scales at <XXX> cost.
+
+User usage patterns
+-------------------
+Another important factor is what usage pattern your users will have. Will they
+all use the JupyterHub at once? Or will their usage be more spread out
```

Comment:
~~the~~  once, such as during a large class workshop? ...Or will users use JupyterHub at different times of day?

The usage patterns and peak load on the system have important...